classes/class-connector.php

```diff
@@ -158,6 +158,12 @@ public function action_links( $links, $record ) {
 	public function log( $message, $args, $object_id, $context, $action, $user_id = null ) {
 		$connector = $this->name;
 
+		/**
+		 * Override the data logged. Returning false to this filter will stop the data from being logged.
+		 * Examples of this filter in use can be found in some of the custom connectors.
+		 *
```

Comment:
<!-- ideas that popped up from reviewing -->
**💡 thought:** @tharsheblows, perhaps it's a good idea to guide devs into a sample.

```suggestion
		 * @see Connector_ACF::log_override()
		 *
```